dotcom-rendering/src/types/front.ts

```diff
@@ -132,8 +134,17 @@ export type DCRContainerPalette =
 // TODO: These may need to be declared differently than the front types in the future
 export type DCRContainerType = FEContainerType;
 
-interface FEMediaAtoms {
+export interface FEMediaAtoms {
+	id: string;
+	activeVersion: number;
+	assets: Array<{ id: string; version: number; platform: string }>;
+	posterImage: { allImages: Image[] };
```

Comment:
I’m not sure if any of these are optional from Frontend’s point of view…

Reply:
I do now: https://github.com/guardian/frontend/blob/0bf69f55a/common/app/model/content/Atom.scala#L158-L169